api/store.js

```diff
@@ -150,6 +150,11 @@ class Store {
           done();
         });
 
+        data.on('end', () => {
+          writeStream.end();
+          done();
+        });
```

Comment:
Some streams emit the `end` event, some emit the `close` event...  It'd be cool if they could all conform to the spec but I guess that's asking too much.